ts/editor/FieldState.svelte

```diff
@@ -11,6 +11,27 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
         display: flex;
         justify-content: flex;
         flex-grow: 1;
-        gap: 6px;
+
+        /* replace these global selectors with "grid-gap: 6px" ASAP */
```

Comment:
I think you mean `flex-gap`, or rather just `gap` (as gap sets both `grid-gap` and `flex-gap`)

Reply:
Oh, I was copy-pasting this comment and didn't notice. Fixing it right now.